src/common/components/Tooltip/Fact/index.js

```diff
@@ -75,10 +113,11 @@ const Fact = ({ data }: FactProps) => {
       break;
 
     case 'BuffConversion':
+      // XXX: String itself should be translated.
```

Comment:
there isn't one atm, but if you add the translation into every text file (under `lib/i18n/texts`), should be fine. we dont need any translations atm really, but just at least something in the texts so someone can replace them at a later date !